recipe-server/normandy/settings.py

```diff
@@ -280,7 +280,7 @@ def RAVEN_CONFIG(self):
     SECURE_PROXY_SSL_HEADER = values.TupleValue()
     SECURE_HSTS_SECONDS = values.IntegerValue(3600)
     SECURE_HSTS_INCLUDE_SUBDOMAINS = values.BooleanValue(True)
-    CSRF_COOKIE_HTTPONLY = values.BooleanValue(True)
+    CSRF_COOKIE_HTTPONLY = values.BooleanValue(False)
```

Comment:
Changing this to `False` since that's the only way to use this in javascript on the front end. 

This should be fine as per the [Django Docs](https://docs.djangoproject.com/en/1.11/ref/settings/#csrf-cookie-httponly):

> Designating the CSRF cookie as HttpOnly doesn’t offer any practical protection because CSRF is only to protect against cross-domain attacks. If an attacker can read the cookie via JavaScript, they’re already on the same domain as far as the browser knows, so they can do anything they like anyway. (XSS is a much bigger hole than CSRF.)